### ISSUE TITLE
[TB] Font setting behavior with slider seems unpredictable

### ISSUE THREAD

**Issue by aaronreed708** [finos/a11y-theme-builder#683]
## Problem/Concern

Here are some inconsistencies that were noticed by Yessel and myself as we looked through the Font Settings

- Font Settings-> base font size field should indicate what is calculated off of the base size.  At the moment it doesn’t seem to affect any of the typography styles.  So the user is left wonder what it does affect.
- Display & Header Styles -> Percent change in Header/.Display sizes slider needs to affect styles predictably and when editing a specific style it should reflect its current outer value.
  - for example, I move the slider to 50 and the first style (Display 1) has a size of 88px.  But if I click on the Edit button, it shows me a font size of 76.48px
- slider value doesn't seem predictable or recoverable.  It should probably be relative to a set, pre-defined value.  So that setting it to 50, for example, at any point in time, will result in the same value for the other Display & Header styles.  Otherwise the user probably needs a way to recover the initial value.
  - currently, the size of the fonts on the Display & Header Styles pane seem to be affected by the percentage value slider and the base font size.  So if the user wants to "recover", they need to know the initial values for both at a minimum.
  - currently, if I have a base font size of 16px and the % slider is set at 50, then Display 1 has a size of 88px.  If I change the value of base font size to 32px and navigate back to the Display & Header Styles, it still shows a size of 88px.  But if I then change the slider value to 51, all of the sudden the font size is way bigger.  So the Display & Header Styles page doesn't reflect the change in base font size until an action is taken.
- Why is Display & Header Styles pane the only one with a slider that affects "child" styles in the pane?  Would this capability be desirable on other panes?

**Comment by evangk6** [comment]
I agree with this - I would like some formally written requirements on how everything font related should work. It is by far one of the most complex atoms and there should be a clear design on how its properties should inter-connect so they can be finalized.

**Comment by anonfaded** [comment]
If the repo owner is looking for frontend developers as open-source contributors, let me know!

**Comment by aaronreed708** [comment]
@anonfaded Welcome!  We are always on the lookout for contributors and would be happy to work with you to get you up to speed on the repo.  I believe that starting with something a little more straight forward than this issue might prove to be a less frustrating way to step into the Theme Builder code.  Would any of the issues #503 - #508 or #518 - #524 interest you?  Organism and Template type issues should be largely frontend and Molecule issues would require frontend and backend modifications.  If you'd like to reach out to me directly, follow the Slack invitation in [Communication](https://github.com/finos/a11y-theme-builder/wiki/Communication).  Thanks!
